### DISCUSSION TITLE
[Choix du statut] Ajouter les SCIC, SCOP, CAPE

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2672] [Idées]
> Pourquoi ne pas évoquer les SCOOP ; les SCIC, le CAPE (couveuses et coopératives) , le portage salarial ?

Le mieux serait de développer le parcours à partir de la question « Faire de l'argent » / « Non lucratif » (à reformuler). 
On peut imaginer ensuite relier le parcours classique pour aider à choisir entre SCOP SARL et SCOP SAS.

**Comment by wiinxt** [comment]
tag : Portage entrepreneurial
